main.js

```diff
@@ -0,0 +1,12 @@
+const source = $('#result-template').html();
+const template = Handlebars.compile(source)
+
+const renderPeople = function (users) {
+    $("#results").empty();
+    const newHTML = template({ data: users });
+    $('#results').append(newHTML);
+}
+
+let peopleArr = $.get('https://jsonplaceholder.typicode.com/users', renderPeople)
+
```

Comment:
- You don't need to assign the value of the `get` to a variable.

- the `peopleArr` will be a promise and not array.